Makefile

```diff
@@ -110,38 +110,38 @@ install:
 	@go install $(LDFLAGS) $(REPO)/$(CMDDIR)/$(GATEWAY)
 
 build:
-	@hack/build $(CLI)
-	@hack/build $(SERVER)
-	@hack/build $(AGENT)
-	@hack/build $(LOGWORKER)
-	@hack/build $(GATEWAY)
+	@hack/build.sh $(CLI)
```

Comment:
I prefer to not add `.sh`. I realize it's a style issue, but I already commented my thoughts on this above. I know everyone isn't 100% consistent out there, including Docker itself. They add the extension under their `hack` directory, and yet have dozens of shell scripts that don't under `hack/make`. Bottom line, unless not having the extension is frustrating everyone with editors that key off the extension for syntax highlighting (I don't have that problem with vim), then let's standardize on leaving it off as I had it originally.